main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.start
+end
+
+main
```

Comment:
- [ ] While you made a big effort working on the entire App, you still have an issue to work on. Your App didn't work correctly I have got the following issue and I recommend checking the files and class names:

<img width="657" alt="Screenshot 2023-09-29 at 5 50 33 PM" src="https://github.com/maths29/OOP-school-library/assets/53347966/1a080d28-0bc0-4c2c-895f-58e3b92aea58">